roles/amq_streams_broker/tasks/service.yml

```diff
@@ -26,6 +26,9 @@
     service_systemd_env_file: "{{ amq_streams_broker_service_config_file }}"
     server_java_opts: "{{ amq_streams_broker_java_opts | default('') }}"
     server_java_heap_opts: "{{ amq_streams_broker_java_heap_opts | default('') }}"
+    server_java_performance_opts: "{{ amq_streams_broker_java_performance_opts | default('') }}"
```

Comment:
using `default('')` will not create the variable with an empty entry? I mean, the idea is only to declare the Kafka variables only if there is a value to add. Maybe I am wrong about Ansible best practices. 

Can you confirm that that definition will not declare the variable at all? Thanks

Reply:
@rpelisse Added this comment :point_up:

Reply:
That'll be great indeed :) CI tells us it's not breaking anything, but we do need to confirm it does what we expect it to do!